source/user-manual/wazuh-dashboard/custom-branding.rst

```diff
@@ -44,6 +44,12 @@ To customize the *global App loading* logo, do the following.
          loadingLogo:
             defaultUrl: "https://domain.org/default-logo.png"
             darkModeUrl: "https://domain.org/dark-mode-logo.png"
+   
+#. Restart the Wazuh dashboard service using this command:
```

Comment:
**suggestion**: I would remove `using this command`.

Reply:
Done [b1434bc](https://github.com/wazuh/wazuh-documentation/pull/7678/commits/b1434bcf4a577d3f46f65cda8905a6a06226cd2f)